README.md

```diff
@@ -56,7 +56,8 @@ In Ubuntu 20.04 and in Mint 20, the VM is built with the following packages:
 ### Building in OSX / Linux:
 
```

Comment:
Could you add a comment in text with something like "We use out-of-source builds:"?